File: source/clibCore/addon_tools/addon_toolConfigureEdit.vb

Option Explicit On
Option Strict On
'

Imports System.Xml
Imports Contensive.Core
Imports Contensive.BaseClasses
'
Namespace Contensive.Addons
    '
    Public Class addon_toolConfigureEditClass
        Inherits Contensive.BaseClasses.AddonBaseClass
        '
        '====================================================================================================
        ''' <summary>
        ''' addon method, deliver complete Html admin site
        ''' </summary>
        ''' <param name="cp"></param>
        ''' <returns></returns>
        Public Overrides Function execute(cp As CPBaseClass) As Object
            Dim result As String = ""
            Try
                'result = GetForm_ConfigureEdit(cp)
            Catch ex As Exception
                cp.Site.ErrorReport(ex)
            End Try
            Return result
        End Function
    End Class
End Namespace

File: source/clibCoreCs/addonCollection_tools/addon_toolConfigureEdit.cs
using System;

//========================================================================
// This conversion was produced by the Free Edition of
// Instant C# courtesy of Tangible Software Solutions.
// Order the Premium Edition at https://www.tangiblesoftwaresolutions.com
//========================================================================

//

using System.Xml;
using Contensive.Core;
using Contensive.BaseClasses;
//
namespace Contensive.Addons
{
	//
	public class addon_toolConfigureEditClass : Contensive.BaseClasses.AddonBaseClass
	{
		//
		//====================================================================================================
		/// <summary>
		/// addon method, deliver complete Html admin site
		/// </summary>
		/// <param name="cp"></param>
		/// <returns></returns>
		public override object Execute(CPBaseClass cp)
		{
			string result = "";
			try
			{
				//result = GetForm_ConfigureEdit(cp)
			}
			catch (Exception ex)
			{
				cp.Site.ErrorReport(ex);
			}
			return result;
		}
	}
}

